Cargo.toml

```diff
@@ -49,5 +49,7 @@ convert_case = "0.6"
 proc-macro2 = "1.0"
 syn = { version = "1.0", features = ["extra-traits", "full"] }
 quote = "1.0"
-serde = { version = "1.0", features = ["derive"] }
+# Prevent downloading and running untrusted and unreproducible executable
+# https://github.com/serde-rs/serde/issues/2538
+serde = { version = "=1.0.171", features = ["derive"] }
```

Comment:
In the end this appears to have been reverted with 1.0.184, lets hold off pinning and monitor if this happens again.

https://github.com/serde-rs/serde/releases/tag/v1.0.184